cmd/test-lambda/main.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import "github.com/aws/aws-lambda-go/lambda"
+
+func hello() (string, error) {
+	return "Hello ƛ!", nil
+}
+
+func main() {
+	// Make the handler available for Remote Procedure Call by AWS Lambda
+	lambda.Start(hello)
```

Comment:
Shamelessly stolen from @imjasonh here: https://github.com/imjasonh/lambda-ko.

Thanks for the pointer!